geoportal/geoportailv3_geoportal/lib/esri_authentication.py

```diff
@@ -0,0 +1,117 @@
+from typing import Optional, Dict
+import json
+from datetime import datetime, timedelta
+import requests
+import urllib
+from collections import namedtuple
+
+
+class ESRITokenException(Exception):
+    pass
+
+
+ResultTuple = namedtuple('ResultTuple', ['data', 'content_type'])
+
+
+def read_request_with_token(url_request, parent_request, log, timeout=15, renew_token=True):
+    result = urllib.request.urlopen(url_request, None, timeout)
+    data = result.read()
+    try:
+        resp = json.loads(data)
+    except Exception as e:
```

Comment:
Maybe use a less generic exception handler ? (JSONDecodeError in this case)

Reply:
@RemiDesgrange 
I remember I tried this, but the JSON Error does not work.
In this case I get a UnicodeDecodeError, but I think it might also be a JSONDecodeError in some cases, so I'm not quite sure if I can use a less generic Exception.
Any Ideas ?